### PR TITLE
Add containerd release notes

### PR DESCRIPTION
**Pull request by bsousaa**
### Proposed changes

Adding release notes of the Containerd Image Store feature released with Docker Desktop 4.13.0.

### CODE REVIEW

File: desktop/containerd/index.md [hunk 1]
```diff
@@ -209,6 +209,19 @@ View Tags on DockerHub to see multi-platform result:
 
 ## Containerd Image Store Release Notes
 
+(2022-10-19)  
+New commands and multiple bug fixes included with Docker Desktop 4.13.0.
+
+### New
+* Fixed: Kubernetes not starting.
```
Comment (aevesdocker) [inline desktop/containerd/index.md:216]: @bsousaa Perhaps it's worth creating a new section called 'Bug fixes' and moving most of these bullet points?

File: desktop/containerd/index.md [hunk 2]
```diff
@@ -209,6 +209,19 @@ View Tags on DockerHub to see multi-platform result:
...
+* Fixed: Kubernetes IN Docker (kind) not starting.
```
Comment (aevesdocker) [inline desktop/containerd/index.md:217]: Also, in keeping with our style guide, and other release notes, it'd be great if we could remove the ':' from all these bullet points and have complete sentences. For example, 'Fixed a bug which caused Kubernetes in Docker (kind) to not start'

File: desktop/containerd/index.md (hunk 1 above)
Comment (bsousaa) [reply desktop/containerd/index.md]: Thanks @aevesdocker ! Done.

File: desktop/containerd/index.md (hunk 2 above)
Comment (bsousaa) [reply desktop/containerd/index.md]: Well noted, done!

**Review by aevesdocker — APPROVED** [review]
Perfect! Thanks @bsousaa

**Pull request merged**